### DISCUSSION TITLE
想打印下图红框区域（会涉及到跨页），试了半天操作DOM，一直不生效，求解决思路

### DISCUSSION THREAD

**Discussion by kanganxi** [slowlyo/owl-admin#116] [FAQ]
<img width="1552" alt="image" src="https://github.com/slowlyo/owl-admin/assets/13616382/67137fc5-bde8-40c7-bfbc-21791d84d881">

**Comment by slowlyo** [comment]
试试这个动作

![image](https://github.com/slowlyo/owl-admin/assets/94038215/dc66d23c-b8b9-4646-bcbc-5fe0525f8e03)

**Reply by kanganxi** [reply]
是个好方法，我能调起打印窗口，但是打印区内容是空白的，不知道啥原因，我再研究研究
